charts/vcluster/Chart.yaml

```diff
@@ -20,5 +20,5 @@ name: vcluster
 sources:
   - https://github.com/loft-sh/vcluster
 type: application
-version: 0.15.2
-appVersion: 0.15.2
+version: 0.1.0
```

Comment:
in the master branch, we usually define it as `0.0.0-dev`, do we want to keep it consistent here?

Reply:
fixed.